pandas/core/window/rolling.py

```diff
@@ -1409,6 +1409,14 @@ def quantile(self, quantile: float, interpolation: str = "linear", **kwargs):
 
         return self._apply(window_func, name="quantile", **kwargs)
 
+    def rank(self, pct: bool = False, **kwargs):
```

Comment:
Bonus points if you could implement `ascending` to match `DataFrame.rank`

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.rank.html#pandas.DataFrame.rank